input-output-hk/cardano-sl
# [CSE-204] Add `maybe` tx inputs

See https://issues.serokell.io/issue/CSE-204

This PR includes few updates of `README` of Explorers frontend, which are not part of `CSE-204`, but just nice to have.

How to test?
1. Build and run Explorer's frontend as described here: https://github.com/input-output-hk/cardano-sl/tree/master/explorer/frontend#build-in-development-mode

2. Run Explorer's mock server
```
$ cardano-sl » stack exec cardano-explorer-mock
```
3. Open http://localhost:3100/tx/b29fa17156275a8589857376bfaeeef47f1846f82ea492a808e5c6155b450e02 and you should see something like this (note that we have "--" for `Nothing` values of `tx inputs` now)

![transaction-maybe](https://user-images.githubusercontent.com/47693/29670402-d34ddd4e-88e6-11e7-9823-2d457218a9f3.png)
